src/functions/template-tags/ical.php

```diff
@@ -44,7 +44,7 @@ function tribe_get_gcal_link( $postId = null ) {
 		$postId    = Tribe__Events__Main::postIdHelper( $postId );
 		$output    = Tribe__Events__Main::instance()->googleCalendarLink( $postId );
 
-		return apply_filters( 'tribe_get_gcal_link', $output );
+		return apply_filters( 'tribe_get_gcal_link', $output, $postId );
```

Comment:
Needn't block merging, but if there was time to add a docblock in here that would be :+1: